tabpy/tabpy_server/common/default.conf

```diff
@@ -1,13 +1,13 @@
 [TabPy]
-TABPY_QUERY_OBJECT_PATH = /tmp/query_objects
-TABPY_PORT = 9004
-TABPY_STATE_PATH = ./tabpy-server/tabpy_server
+# TABPY_QUERY_OBJECT_PATH = /tmp/query_objects
```

Comment:
why even specify a default.conf if it's all commented out and we already have documentation on how to create your own custom.conf?

Reply:
I simply didn't want to add change to completely remove the default config file. If we do that we will need some template for the config anyway. 

Probably should do with some next improvement.